agir/front/components/authentication/Connexion/LoginMailEmpty.js

```diff
@@ -55,7 +55,10 @@ const LoginMailEmpty = () => {
     next = new URLSearchParams(location.search).get("next");
 
   const handleInputChange = (e) => {
-    setEmail(e.target.value);
+    const value = e.target.value;
+    const i = value.indexOf(" ");
+    if (-1 !== i) value = value.substr(0, value.indexOf(" "));
+    setEmail(value);
```

Comment:
```suggestion
    setEmail(e.target.value.trim());
```

https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Global_Objects/String/Trim